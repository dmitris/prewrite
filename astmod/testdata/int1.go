// package doc comment for github.com/foo/bar
package bar

import _ "go.corp.example.com/abc/xyz"

func Bar() {
	// unrelated comment in func
	println("Hello, World! (from github.com/foo/bar)")
}
