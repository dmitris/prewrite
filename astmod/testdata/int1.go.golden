// package doc comment for github.com/foo/bar
package bar

import _ "go.brandnewcorp.com/abc/xyz"

func Bar() {
	// unrelated comment in func
	println("Hello, World! (from github.com/foo/bar)")
}
